crates/holochain_state/src/mutations.rs

```diff
@@ -337,7 +334,6 @@ pub fn insert_op_lite_when(
         }
         DhtOpLite::Warrant(op) => {
             let _warrant_hash = op.warrant().to_hash();
```

Comment:
Same as above, it's always used now.
```suggestion
            let warrant_hash = op.warrant().to_hash();
```

Reply:
https://github.com/holochain/holochain/pull/5418/commits/fa7588016bd40ccc83e70e44ad43b09d1168dd00